java/vector/src/main/java/org/apache/arrow/vector/BitVectorHelper.java

```diff
@@ -0,0 +1,42 @@
+package org.apache.arrow.vector;
+
+import io.netty.buffer.ArrowBuf;
+
+public class BitVectorHelper {
```

Comment:
This class no public static methods?

Reply:
I found a public util class that has no public methods to be weird. Maybe making this class package private if that's the intention?

Reply:
Sounds good.

Reply:
Done.